freetextresponse/freetextresponse.py

```diff
@@ -242,6 +242,16 @@ def student_view(self, context=None):
         )
         return fragment
 
+    def max_score(self):
+        """
+        Returns the configured number of possible points for this component.
+        Arguments:
+            None
+        Returns:
+            float: The number of possible points for this component
+        """
+        return self.weight if self.has_score else None
```

Comment:
Shouldn't this always return the _raw_ max score, `Credit.full.value`? The `has_score` attribute is defaulted to True so the boolean check is superfluous.